src/scss/foundations/containers.scss

```diff
@@ -0,0 +1,68 @@
+@use "../lib";
+@use "breakpoints";
+
+/**
+ * The gutter of container
+ *
+ * @type Number
+ * @access private
+ */
+$_gutter: lib.rem(20px);
+
+/**
+ * Fluid dimension for small size
+ *
+ * @type Number
+ * @access private
+ */
+$_fluidWidth: calc(100vw - (#{$_gutter} * 2));
+
+/**
+ * The breakpoints of Pyxis
+ *
+ * @type Map
+ * @access private
+ */
+$_containers: (
+  base: $_fluidWidth,
+  xxsmall: $_fluidWidth,
+  xsmall: $_fluidWidth,
+  small: $_fluidWidth,
+  medium: lib.rem(960px),
+  large: lib.rem(1140px),
+  xlarge: lib.rem(1370px),
+);
+
+/**
+ * Create atom classes for containers
+ *
+ * @access public
+ */
+@mixin createAtoms() {
```

Comment:
Più che mettere tutto in atoms, farei un mixin pubblico `container()`. Questo può essere usato via `atoms` solo se viene utilizzato `createAtoms`. Ricordati che quest'ultima cosa è opzionale quindi deve essere possibile anche fare una cosa del tipo: 
```scss
.my-wrapper {
  @include container();
}
```

Reply:
Idealmente potremmo fare in modo anche che 
```scss
@mixin container($fromBreakpoint: null, $toBreakpoint: null) {
 // Qui definisco le regole avendo l'opzione di settare il container ad una dimensione fissa tra 
 // due breakpoint e facendogli occupare il 100% della width quando si è fuori da essi.
}
```

Reply:
Giusto, pensavo solo alle classi come nel vecchio pyxis, ma meglio avere anche il mixin 👌

Reply:
@alessiosantangelo Aggiunto il mixin, ma come anche per le classi ho preferito mettere solo il `$fromBreakpoint` (con valore impostato su `base` per semplicità. Il `toBreakpoint` non l'abbiamo mai utilizzato e mi sembra poco utile

Reply:
@emanuelemacri senza mettere il `to` non c'è la possibilità di fare un container _fluido fino a bpSmall, boxato da bpSmall a bpMedium, e fluido da bpMedium in poi_.

In questo caso secondo me dovrebbe essere concesso. Anche perchè scrivere`@include container(large, small);` seppur sbagliato dal punto di vista dell'utilizzo, produrrebbe codice "morto" in quanto `min-width e max-width` sarebbero espresse al contrario. 

Possiamo invece inserire nel mixin un po' di controlli del tipo "il from deve essere maggiore del to" e così via.

Reply:
@alessiosantangelo pushato la modifica e aggiunto l'example alla documentazione